projects/adf-office-services-ext/src/lib/evaluators.spec.ts

```diff
@@ -220,6 +233,7 @@ describe('evaluators', () => {
 
     it('should return [false] if current user is not lock owner', () => {
       const context: any = {
+        aosPlugin: true,
```

Comment:
do we really need to add it everywhere?

Reply:
yeah, maybe worth having it in the "beforeEach" section